preprocessing/cameras/create_missing_video_segments_from_raw_videos.py

```diff
@@ -0,0 +1,278 @@
+    raw_videos_for_cam = camera_raw_to_segment[f"cam{camera_index}"]
+    camera_was_rotated = camera_was_rotated_map[f"cam{camera_index}"]
+
+    # Then we iterate over all the raw videos for the given camera, each will be referred
+    # with a specific video_index, when generating the output videos. For example vid{video_index}-seg1.mp4
+    for video_index, raw_video_file_basename in camera_raw_to_segment[
+        f"cam{camera_index}"
+    ].items():
+        if video_index < 3:
+            # For the MINGLE experiments, we want to annotate video segments AFTER vid3-seg6.
+            # We don't get specific into the segment, because the logic below skips if the desired
+            # segment video already exists anyway.
+            continue
+
+        raw_video_file_path = (
+            RAW_VIDEOS_FOLDER_IN_STAFF_BULK
+            / f"cam{camera_index:02}"
+            / raw_video_file_basename
+        )
+        if camera_was_rotated:
+            # Some cameras were rotated, and we make sure that the rotated version exists
+            raw_rotated_file_basename = raw_video_file_basename.replace(
+                ".MP4", "_rot.MP4"
+            )
+            raw_rotated_video_file_path_in_staff_bulk = (
+                RAW_VIDEOS_FOLDER_IN_STAFF_BULK
+                / f"cam{camera_index:02}"
+                / raw_rotated_file_basename
+            )
+
+            if raw_rotated_video_file_path_in_staff_bulk.exists():
+                raw_video_file_path = raw_rotated_video_file_path_in_staff_bulk
+            else:
+                raw_rotated_video_file_path_in_local = (
+                    VIDEO_SEGMENTS_FOLDER_IN_LOCAL
+                    / "raw_overhead"
+                    / f"cam{camera_index:02}"
+                    / raw_rotated_file_basename
+                )
+
+                if not raw_rotated_video_file_path_in_local.exists():
+                    raw_rotated_video_file_path_in_local.parent.mkdir(
+                        parents=True, exist_ok=True
+                    )
+                    # Reference from /mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/raw/video/overhead/cam04/videoRot.sbatch
+                    cmd = [ 
+                        "ffmpeg",
+                        "-i",
+                        str(raw_video_file_path),
+                        "-c",
+                        "copy",
+                        "-metadata:s:v:0",
+                        'rotate=0',
+                        str(raw_rotated_video_file_path_in_local),
+                    ]
+                    print("================= ROTATING =======================")
+                    subprocess_run_with_guardfile(
+                        cmd,
+                        raw_rotated_video_file_path_in_local.with_suffix(
+                            ".isincomplete.txt"
+                        ),
+                    )
+
+                raw_video_file_path = raw_rotated_video_file_path_in_local
```

Comment:
`vid5-seg1-scaled-denoised` was not rotated. I fixed it manually but we need to double check why that happened for that segment